### PR TITLE
Cloud database

### PR DESCRIPTION
**Pull request by PaulFenton**
1. added cloud MySQL database implementation
2. Updated database creation script

### CODE REVIEW

**Comment by darioalessandro** [comment]
Pr looks good, I recommend deleting the credentials and url. Instead please pass those as environment variables, take a look at how I pass the oauth client Id to the frontend

File: A2-2015/A2-Rewrite/database.sql [hunk 1]
```diff
@@ -1,25 +1,14 @@
...
-/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
-
-/*!40101 SET @saved_cs_client     = @@character_set_client */;
-/*!40101 SET character_set_client = utf8 */;
-
-USE inventory_v2;
+USE eep_operations;
 
 DROP TABLE IF EXISTS `auth_logs`;
+DROP TABLE IF EXISTS `product`;
```
Comment (darioalessandro) [inline A2-2015/A2-Rewrite/database.sql:7]: 👍

File: A2-2015/A2-Rewrite/backend/index.js [hunk 2]
```diff
@@ -115,6 +114,7 @@ app.post('/auth/log/:type',jsonParser,async function (req, res) {
 });
 
 app.get('/products',async  function (req, res) {
+    console.log("username: ", process.env.EEP_DATABASE_ADMIN_NAME, "password: ", process.env.EEP_DATABASE_PASSWORD);
```
Comment (darioalessandro) [inline A2-2015/A2-Rewrite/backend/index.js:117]: 👍

File: README.md [hunk 3]
```diff
@@ -40,14 +40,23 @@ https://www.codementor.io/mercurial/how-to-install-node-js-on-macos-sierra-mphz4
 nvm install 8.10.0
 ```
 
-#### 4. run the backend application
+#### 4. set backend database credentials
```
Comment (darioalessandro) [inline README.md:43]: Can we add the url as a env var as well? I would like to test locally with my MySql instance

File: README.md [hunk 4]
```diff
@@ -40,14 +40,23 @@ https://www.codementor.io/mercurial/how-to-install-node-js-on-macos-sierra-mphz4
...
+
+```bash
+export EEP_DATABASE_ADMIN_NAME='eep_admin'
```
Comment (darioalessandro) [inline README.md:46]: are these the actual credentials? can you send them as an email instead and just leave placeholders here?

**Review by darioalessandro — COMMENTED** [review]
LGTM I just have a request regarding exposing the db as a parameter as well and a question regarding the db credentials.

File: README.md (hunk 3 above)
Comment (PaulFenton) [reply README.md:43]: I can... there are several other differences to the config for the mongo db.. how do you want to manage all of that?

File: README.md (hunk 4 above)
Comment (PaulFenton) [reply README.md:46]: yes there are actual credentials ... I can email

**Review by darioalessandro — APPROVED** [review]
LGTM we will revisit the password stuff later.

**Pull request merged**